cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -211,16 +242,38 @@ class ProfilesCreateView extends Component {
                     >
                       {property.label}
                     </strong>
+                    {
+                      property.required ?
+                        <span className="required-marker text-danger">*</span>
+                      :
+                        null
+                    }
+                    {
+                      property.description ?
+                        <UncontrolledTooltip
+                          placement="right"
+                          delay={0}
+                          target={uniqueId}
```

Comment:
I'm seeing something like `undefined-projectId` for these ids. I think it's because at https://github.com/caskdata/cdap/pull/10072/files#diff-0cb655c9bf69ab929ea591f4afffdb3aR235 you're defining `uniqueId = ${group.name}-${property.name}`, but there's no `group.name` that I can see in the json spec, only `group.label`